pocs/tests/utils/test_utils.py

```diff
@@ -9,12 +9,34 @@
 from pocs.utils import listify
 from pocs.utils import load_module
 from pocs.utils import CountdownTimer
-from pocs.utils.error import NotFound
+from pocs.utils import error
 from pocs.camera import list_connected_cameras
 
 
+def test_error(capsys):
+    with pytest.raises(Exception) as e_info:
+        raise error.PanError(msg='Testing message')
+
+    assert str(e_info.value) == 'PanError: Testing message'
+
+    with pytest.raises(Exception) as e_info:
+        raise error.PanError()
+
+    assert str(e_info.value) == 'PanError'
+
+    with pytest.raises(SystemExit) as e_info:
```

Comment:
As a reader, this is definitely odd. Why should it be this way? It seems that the only thing that PanError(exit=True) provides is a log.error followed by system.exit. It seems oddly hidden in raising an exception.

Reply:
Yes, I'm not sure this is how we should be counting on the system exit. This is a bit of a legacy and probably could be fixed up. I think it is only used in one location.

Reply:
Made #676